tests/integration/kubernetes/gha-run.sh

```diff
@@ -431,6 +436,33 @@ function cleanup_snapshotter() {
 	echo "::endgroup::"
 }
 
+function reset_k8s_images(){
+	# we are encountering the issue (https://github.com/kata-containers/kata-containers/issues/8407)
+	# with containerd on CI is likely due to the content digest being missing from the content store,
+	# which can happen when switching between different snapshotters.
+	# To help sort it out, we now clean up or fetch related images in k8s.io namespace.
+	ctr_args=""
+	if [ "${KUBERNETES}" = "k3s" ]; then
+		ctr_args="--address /run/k3s/containerd/containerd.sock "
+	fi
+	ctr_args+="--namespace k8s.io"
+	ctr_command="sudo -E ctr ${ctr_args}"
+	pause_images=$(${ctr_command} i ls | grep 'pause' | awk '{print $1}')
+	for image in $pause_images; do
+		${ctr_command} content fetch "$image"
+	done
+	# remove related images in k8s.io namespace
+	images_to_remove=(
+		"quay.io/sjenning/nginx:1.15-alpine"
+		"quay.io/prometheus/busybox:latest"
+		"quay.io/confidential-containers/test-images:largeimage"
+	)
+
+	for image in "${images_to_remove[@]}"; do
+		${ctr_command} i remove "$image" || true
```

Comment:
I'm not a `ctr` expert, but I suppose this `remove` could fail for a valid reason, but the `|| true` would hide it. I guess you have the short-circuit logic so that if the image doesn't exist, you'll still continue normally? Maybe it's better to do something like (pseudocode):
```
for image in images; do
  if image exists; then
    ctr remove image # do not use || true here
  fi
done
```

Reply:
Good point. I have fixed it. Thanks.